static_files/index.html

```diff
@@ -51,15 +51,15 @@
 
     <!-- Main Content -->
 
-    <main>
+    <main class="Main">
```

Comment:
Apparently the ".Main" class is not needed in this case.
```suggestion
    <main>
```